tests/test_azure.py

```diff
@@ -518,6 +518,7 @@ def test_upload_to_container_blob_from_storage(self,
         mock_blob.exists.assert_called_once()
         mock_blob.upload_blob.assert_called_once_with(
             data=ANY,
+            length=0,
```

Comment:
Maybe we can try to write some data to tmpfile and check expected legth
```
>>> tmpfile.seek(1024)
1024
>>> tmpfile = NamedTemporaryFile()
>>> tmpfile.seek(1020)
1020
>>> tmpfile.write(b'1234')
4
>>> tmpfile.flush()
```

Reply:
Sure!

Reply:
Fixed

Reply:
tyvm